lms/djangoapps/discussion/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""
+Custom exceptions raised by Discussion API.
+"""
+
+
+class DiscussionHiddenFromUserException(BaseException):
```

Comment:
:100: for custom exceptions

Suggestion: Call this `TeamDiscussionHiddenFromUserException`, because it's handled with teams-specific messaging.